diff -Naur buildroot/package/x11r7/xdriver_xf86-video-fbdev/xdriver_xf86-video-fbdev.hash buildroot/package/x11r7/xdriver_xf86-video-fbdev.new/xdriver_xf86-video-fbdev.hash
--- buildroot/package/x11r7/xdriver_xf86-video-fbdev/xdriver_xf86-video-fbdev.hash	2020-03-21 19:02:31.082590582 +0100
+++ buildroot/package/x11r7/xdriver_xf86-video-fbdev.new/xdriver_xf86-video-fbdev.hash	2020-03-21 17:07:45.562886376 +0100
@@ -1,2 +1,2 @@
 # From http://lists.x.org/archives/xorg-announce/2013-September/002324.html
-sha256	9dd4b326498223abbfdf786089a46ea3db4fa6bbd341308eb48a9e00bc3fd51b	xf86-video-fbdev-0.4.4.tar.bz2
+sha256	dcc3d85f378022180e437a9ec00a59b6cb7680ff79c40394d695060af2374699	xf86-video-fbdev-0.5.0.tar.bz2
diff -Naur buildroot/package/x11r7/xdriver_xf86-video-fbdev/xdriver_xf86-video-fbdev.mk buildroot/package/x11r7/xdriver_xf86-video-fbdev.new/xdriver_xf86-video-fbdev.mk
--- buildroot/package/x11r7/xdriver_xf86-video-fbdev/xdriver_xf86-video-fbdev.mk	2020-03-21 19:02:31.082590582 +0100
+++ buildroot/package/x11r7/xdriver_xf86-video-fbdev.new/xdriver_xf86-video-fbdev.mk	2020-03-21 17:07:59.098275749 +0100
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-XDRIVER_XF86_VIDEO_FBDEV_VERSION = 0.4.4
+XDRIVER_XF86_VIDEO_FBDEV_VERSION = 0.5.0
 XDRIVER_XF86_VIDEO_FBDEV_SOURCE = xf86-video-fbdev-$(XDRIVER_XF86_VIDEO_FBDEV_VERSION).tar.bz2
 XDRIVER_XF86_VIDEO_FBDEV_SITE = http://xorg.freedesktop.org/releases/individual/driver
 XDRIVER_XF86_VIDEO_FBDEV_LICENSE = MIT
