diff --git a/arch/arm/kernel/return_address.c.bak b/arch/arm/kernel/return_address.c
index 0697db6..c96c28c 100644
--- a/arch/arm/kernel/return_address.c.bak
+++ b/arch/arm/kernel/return_address.c
@@ -58,10 +58,12 @@ void *return_address(unsigned int level)
 
 #else /* if defined(CONFIG_FRAME_POINTER) && !defined(CONFIG_ARM_UNWIND) */
 
+/*
 void *return_address(unsigned int level)
 {
 	return NULL;
 }
+*/
 
 #endif /* if defined(CONFIG_FRAME_POINTER) && !defined(CONFIG_ARM_UNWIND) / else */
 
